WeakAuras/Prototypes.lua

```diff
@@ -3459,18 +3459,18 @@ Private.event_prototypes = {
         end
       },
       {
-        name = "sourceRaidMarkIndex",
-        display = WeakAuras.newFeatureString .. L["Source Raid Mark Index"],
+        name = "sourceMarkIndex",
```

Comment:
Don't rename them.

Reply:
Since these names would only appear in the tooltip, and only be interacted with in text replacements, the long names were really getting out of hand. But if you're happy with them then that's fine.

Reply:
Also code, which we sometimes can't avoid to break. Also if you rename it, you need to write the modernize code to not break text replacement.

Reply:
Those are only new values being renamed, not ones that existed before